en/cpp/quickstart.md

```diff
@@ -1,72 +1,52 @@
 # QuickStart
```

Comment:
Rename to C++ QuickStart (this is what gets displayed when you do a page search)